src/SignupWithMobile.js

```diff
@@ -40,14 +42,30 @@ const defaultStyles = StyleSheet.create({
     marginTop: 27,
     fontSize: 24,
   },
+  countryPickerContainer: {
+    width: 108,
+  },
+  countryPicker: {
+    borderBottomColor: "rgb(170, 170, 170)",
+    borderBottomWidth: 1,
+  },
+  countryPickerText: {
+    paddingVertical: 11,
+    fontSize: 16,
+  },
   mobileNumberContainer: {
+    marginLeft: 15,
     flex: 1,
-    marginTop: 8,
   },
   mobileNumberText: {
+    // TextInput will have default padding on Android, and it can't be remove.
```

Comment:
Actually it can, by setting paddingVertical to 0

Reply:
After I recheck again, I found out the problem is Text and TextInput has difference default height.

Reply:
The difference in height is due to the fact that TextInput on Android has intrinsic vertical padding which Text does not have.